avpipe.go

```diff
@@ -1459,15 +1459,15 @@ func Probe(params *XcParams) (*ProbeInfo, error) {
 
 		rot := float64(probeArray[i].side_data.display_matrix.rotation)
 		if rot != 0.0 {
-			probeInfo.StreamInfo[i].SideDataList = make([]interface{}, 1)
+			probeInfo.StreamInfo[i].SideData = make([]interface{}, 1)
 			displayMatrix := SideDataDisplayMatrix{
 				Type:       "Display Matrix",
 				Rotation:   rot,
 				RotationCw: float64(probeArray[i].side_data.display_matrix.rotation_cw),
 			}
-			probeInfo.StreamInfo[i].SideDataList[0] = displayMatrix
+			probeInfo.StreamInfo[i].SideData[0] = displayMatrix
 		} else {
-			probeInfo.StreamInfo[i].SideDataList = make([]interface{}, 0)
+			probeInfo.StreamInfo[i].SideData = make([]interface{}, 0)
```

Comment:
nit: Is there a reason to instantiate this here as opposed to just leaving it `nil`? Doesn't matter much regardless, just a note.

Reply:
This is not perfect, and must be rewritten with a good framework when there is more side data.

Reply:
Yeah, that makes sense. 

By the way, just in case you're not familiar with it (I brought it from my last company, not sure how widespread it is), anything that I preface with "nit:" is very much so a non-blocking change / question.